apps/src/utils.js

```diff
@@ -171,17 +175,15 @@ export function randomKey(obj) {
  *
  * @returns {string} RFC4122-compliant UUID
  */
-export function createUuid() {
+export function createUuid():string {
   return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
     var r = Math.random()*16|0, v = c === 'x' ? r : (r&0x3|0x8);
     return v.toString(16);
   });
 }
 
 export function fireResizeEvent() {
-  var ev = document.createEvent('Event');
-  ev.initEvent('resize', true, true);
-  window.dispatchEvent(ev);
+  window.dispatchEvent(new CustomEvent('resize', {bubbles: true, cancelable: true}));
```

Comment:
Creating events the way is deprecated and as such, flow type does not include the deprecated functions in its DOM api type definitions. Since we aren't supporting any browsers that only use the deprecated api, this change should be a no-op.